src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -780,11 +784,11 @@ private void createBom(Bom bom, MavenProject mavenProject) throws ParserConfigur
             final BomXmlGenerator bomGenerator = BomGeneratorFactory.createXml(schemaVersion(), bom);
             bomGenerator.generate();
             final String bomString = bomGenerator.toXmlString();
-            final File bomFile = new File(mavenProject.getBasedir(), "target/" + outputName + ".xml");
-            getLog().info(String.format(MESSAGE_WRITING_BOM, "XML", bomFile.getAbsolutePath()));
+            final File bomFile = new File(mavenProject.getBasedir(), format("%s/%s.xml", outputDirectory, outputName));
```

Comment:
with previous comment change of default value, the `mavenProject.getBasedir()` part is not useful any more

Reply:
> with previous comment change of default value, the `mavenProject.getBasedir()` part is not useful any more

Can you please confirm this?  `${project.build.directory}` is still relative the the project root

Reply:
uh, you're right: https://maven.apache.org/ref/3.8.4/maven-model/maven.html#class_build

Reply:
What needs to change here? Anything?

Reply:
Any guidance on when this change will get implemented? This feature would be a very useful addition to this project.